sycl/doc/EnvironmentVariables.md

```diff
@@ -77,6 +55,49 @@ Assuming a filter has all three elements of the triple, it selects only those de
 
 Note that all device selectors will throw an exception if the filtered list of devices does not include a device that satisfies the selector. For instance, `SYCL_DEVICE_FILTER=cpu,level_zero` will cause `host_selector()` to throw an exception. `SYCL_DEVICE_FILTER` also limits loading only specified plugins into the SYCL RT. In particular, `SYCL_DEVICE_FILTER=level_zero` will cause the `cpu_selector` to throw an exception since SYCL RT will only load the `level_zero` backend which does not support any CPU devices at this time. When multiple devices satisfy the filter (e..g, `SYCL_DEVICE_FILTER=gpu`), only one of them will be selected.
 
+## Controlling DPC++ Level Zero Plugin
+
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| `SYCL_ENABLE_PCI` | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 0. |
+| `SYCL_PI_LEVEL_ZERO_DISABLE_USM_ALLOCATOR` | Any(\*) | Disable USM allocator in Level Zero plugin (each memory request will go directly to Level Zero runtime) |
+| `SYCL_PI_LEVEL_ZERO_TRACK_INDIRECT_ACCESS_MEMORY` | Any(\*) | Enable support of the kernels with indirect access and corresponding deferred release of memory allocations in the Level Zero plugin. |
+
+`(*) Note: Any means this environment variable is effective when set to any non-null value.`
+
+## Tools variables
+
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| `INTEL_ENABLE_OFFLOAD_ANNOTATIONS` | Any(\*) | Enables ITT Annotations support for SYCL runtime. This variable should only be used by tools, that support ITT Annotations. |
+
+`(*) Note: Any means this environment variable is effective when set to any non-null value.`
+
+## Debugging variables for DPC++ Runtime
+
+**Warning:** the environment variables described below are used for
+development and debugging of DPC++ compiler and runtime. Their semantics are
+subject to change. Do not rely on these variables in production code.
```

Comment:
Maybe it would stop more people from using these vars if we say that they may have bugs that we may refuse to fix. :)

Reply:
They're not bugs, they're undefined behavior